content/en/docs/installation/guide/operator-k3s.md

```diff
@@ -0,0 +1,297 @@
+* You are familiar with [Kubernetes Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/), which use custom resources to manage applications and their components
+* You understand the concept of [managing Kubernetes resources using manifests](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/)
+
+## Create an AWS EC2 instance
+
+* Configuration:
+
+  * Ubuntu Server 18.04 LTS (HVM), SSD Volume Type; 64-bit (x86)
+  * Minimum 2 vCPUs
+  * Minimum 8 GB of memory
+  * Minimum of 50 GB of storage
+  * Public IP
+
+## Install K3s
+
+SSH into your VM and run the following command to install the latest version of K3s:
+
+```bash
+curl -sfL https://get.k3s.io | sh -s - --write-kubeconfig-mode 644
+```
+
+Terminal output is similar to:
+
+```bash
+[INFO]  Finding release for channel stable
+[INFO]  Using v1.18.6+k3s1 as release
+[INFO]  Downloading hash https://github.com/rancher/k3s/releases/download/v1.18.6+k3s1/sha256sum-amd64.txt
+[INFO]  Downloading binary https://github.com/rancher/k3s/releases/download/v1.18.6+k3s1/k3s
+[INFO]  Verifying binary download
+[INFO]  Installing k3s to /usr/local/bin/k3s
+[INFO]  Creating /usr/local/bin/kubectl symlink to k3s
+[INFO]  Creating /usr/local/bin/crictl symlink to k3s
+[INFO]  Creating /usr/local/bin/ctr symlink to k3s
+[INFO]  Creating killall script /usr/local/bin/k3s-killall.sh
+[INFO]  Creating uninstall script /usr/local/bin/k3s-uninstall.sh
+[INFO]  env: Creating environment file /etc/systemd/system/k3s.service.env
+[INFO]  systemd: Creating service file /etc/systemd/system/k3s.service
+[INFO]  systemd: Enabling k3s unit
+Created symlink /etc/systemd/system/multi-user.target.wants/k3s.service → /etc/systemd/system/k3s.service.
+[INFO]  systemd: Starting k3
+```
+
+## Create an S3 bucket
+
+Spinnaker's Front50 service needs access to an S3 bucket, so create an S3 bucket with a globally unique name. See the [Creating a bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html) page in the _Amazon Simple Storage Service_ docs for how to create a bucket and naming constraints.
+
+On the **Configure options** screen, select **Versioning** and **Default encryption**.
+
+![ Create bucket - configure options](/images/installation/guide/create-bucket-config-options.jpg)
+
+On the **Set permissions** screen, select **Block all public access**.
+
+Create your bucket.
+
+## Create an IAM Role
+
+Create an IAM Role that you will attach to your EC2 instance. Calls to s3 will use this role to get credentials for the requests. You can read more about IAM Roles in AWS' [AWS Identity and Access Management](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html) guide.
+
+From the **Services** menu, select **IAM**, which is in the **Security, Identity, & Compliance** section.
+
+![ Select IAM](/images/installation/guide/selectIAM.png)
+
+Select the **Roles** section.
```

Comment:
Is there a reason you chose not to do these steps as an ordered list?

Reply:
easier spacing?